public/js/app/util/helpers.js

```diff
@@ -7,12 +7,8 @@ export const combinedCourseName = (courseCode, course, langAbbr) => {
   const { credits, title } = course
   let { creditUnitAbbr = '' } = course
   creditUnitAbbr = typeof creditUnitAbbr === 'object' ? creditUnitAbbr[langAbbr] : creditUnitAbbr
-  const creditsStandard = credits.toString().indexOf('.') < 0 ? credits + '.0' : credits
-  const localeCredits =
-    langAbbr === 'sv' ? creditsStandard.toLocaleString('sv-SE') : creditsStandard.toLocaleString('en-US')
   const creditUnit = langAbbr === 'sv' ? creditUnitAbbr : 'credits'
-
-  const courseName = `${courseCode} ${title[langAbbr]} ${localeCredits} ${creditUnit}`
+  const courseName = `${courseCode} ${title} ${credits.number} ${creditUnit}`
```

Comment:
We added this `formattedWithUnit` on course which should contains credits and unit formatted correctly for given language. So I think it's better to use that one.
```
const courseName = `${courseCode} ${title} ${credits.formattedWithUnit}`
```